packages/admob/lib/index.d.ts

```diff
@@ -237,6 +237,11 @@ export namespace FirebaseAdMobTypes {
      */
     SMART_BANNER: 'SMART_BANNER';
 
+    /**
+     * A (next generation) dynamically sized banner that is full-width and auto-height.
```

Comment:
If there is only an iOS implementation - should mention it is iOS only?

```suggestion
     * A (next generation) dynamically sized banner that is full-width and auto-height.
     *
     * iOS only
```